k2/csrc/torch_api.h

```diff
@@ -0,0 +1,75 @@
+/**
```

Comment:
Only this header file is visible to users.

We can wrap other classes of k2 when needed.

Reply:
@danpovey 

If you find this approach is feasible, I will continue with it.